iib/workers/tasks/utils.py

```diff
@@ -141,3 +144,30 @@ def run_cmd(cmd, params=None, exc_msg=None, cmd_repr=None):
         raise IIBError(exc_msg)
 
     return response.stdout
+
+
+def request_logger(request_id_index=2):
```

Comment:
I can't think of a nicer way to grab the request ID. I made this configurable to hopefully reduce some of the magic.

Reply:
If you're not opposed to magic, we did something similar in MBS to find a specific argument value of a handler:
https://pagure.io/fm-orchestrator/blob/master/f/module_build_service/scheduler/route.py#_33

Reply:
I'll look into this. It's better magic than what I have :)

Reply:
Done.